INN/umbrella-ipbs
# Button styles

## Changes

This pull request makes the following changes:

- Adds `.btn` and `.btn-blue` mixins
- Applies the `.btn` mixin to Gutenberg button blocks, but doesn't override the colors set by the color picker
- Swaps `ipbs_block_colors_style` to use `wp_head` instead of `wp_print_styles` because of https://make.wordpress.org/core/2011/12/12/use-wp_enqueue_scripts-not-wp_print_styles-to-enqueue-scripts-and-styles-for-the-frontend/

Red:
![Screen Shot 2020-01-28 at 3 40 48 PM](https://user-images.githubusercontent.com/18353636/73303949-56ee5e80-41e5-11ea-8842-39b1ae6905d4.png)

Blue:
![Screen Shot 2020-01-28 at 3 41 04 PM](https://user-images.githubusercontent.com/18353636/73303948-56ee5e80-41e5-11ea-93f0-2fabc3ad9c3a.png)

## Why



For #63

## Testing/Questions

Features that this PR affects:

- Buttons!


Questions that need to be answered before merging:


Steps to test this PR:

1. Apply the `.btn` and `.btn-blue` mixins to something, such as the `More` widget button links
```
.widget {
    .morelink {
        a {
            .btn;
        }
    }
}
```
2. View on frontend and verify it looks ok
3. Add a Gutenberg button block to a post with unique colors
4. View it on the frontend and verify the colors are the ones you chose but all of the other button styling persists
